.circleci/config.yml

```diff
@@ -306,7 +307,12 @@ jobs:
           working_directory: packages/contracts-bedrock
       - run:
           name: test and generate coverage
-          command: yarn coverage:lcov
+          command: |
+            forge --version
+            circleci tests glob "contracts/test/*.sol" "contracts/test/*/*.sol" | circleci tests split | tr "\n" "|" | tee splits.txt
+            cat splits.txt | tr "|" "\n"
+            forge test --list --match-path=$(cat splits.txt)
+            yarn coverage:lcov $(cat splits.txt)
```

Comment:
This is going to run the tests twice, which is why previously we've just relied on the coverage command to run the tests.